internal/audit/hub/conf.go

```diff
@@ -40,6 +43,8 @@ type Conf struct {
 type IngestConf struct {
 	// MaxBatchSize defines the max number of log entries to send in each Ingest request.
 	MaxBatchSize uint `yaml:"maxBatchSize" conf:",example=32"`
+	// MaxBatchSizeBytes defines the max cumulative size in bytes for a batch of log entries.
+	MaxBatchSizeBytes uint `yaml:"maxBatchSizeBytes" conf:",example=2097152"`
```

Comment:
I don't think it's useful to have this configurable by the end users because the limit is ultimately dictated by us. It could be a hidden config such as [`engine.numWorkers`](https://github.com/cerbos/cerbos/blob/380d8f6553a2dccfbd52b8a91823d06231799485/internal/engine/conf.go#L27) that we can use as an escape hatch though.